### ISSUE TITLE
Add missing documentation for TabNav component

### ISSUE THREAD

**Issue by antictor** [Royal-Navy/design-system#994]
Would it be possible to integrate Tabsets with React Router or be able to store the currently selected tab in the url.

The reason for this is 2 fold. There is a situation where I want to have a url to get to a page and have the non-default tab selected. The other situation is I may have the 2nd or 3rd tab open and refresh the page, I want the tab that was selected to remain selected.

**Comment by antictor** [comment]
This is going to be important for my project. I'm going to fork the component code in my repo to get this working for now to unblock me

**Comment by antictor** [comment]
Looks like there is a component for this already that just isn't documented or exported, TabNav but it is missing documentation and is not exported

**Comment by hxltrhuxze** [comment]
Issue has been repurposed.

**Comment by hxltrhuxze** [comment]
Closing as this ticket is no longer relevant
